website/docs/reference/resource-configs/batch_size.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "batch_size"
+id: "batch-size"
+sidebar_label: "batch_size"
+resource_types: [models]
+description: "dbt uses `batch_size` to detrmine how large batches are when running a microbatch incremental model."
+datatype: hour | day | month | year
+---
+
+Available in dbt Cloud Versionless and dbt Core v1.9 and higher.
+
+## Definition
+
+Set the `batch_size` either `hour`, `day`, `month`, or `year`. You can configure `batch_size` for a [model](/docs/build/models) in your `dbt_project.yml` file, property YAML file, or config block.
+
+## Examples
+
+<Tabs> 
+
+<TabItem value="model" label="Models">
```

Comment:
@QMalcolm Do we consider this a "work in progress"? There's a single tab item in this section.

Reply:
More tabs are not expected. I just didn't know what the format should be if it's only applicable to `Models`, should it just be a sub headding?